.storybook/preview.tsx

```diff
@@ -55,7 +57,7 @@ const withThemeDocs: DecoratorFn = ({children, context}) => {
         <ThemeProvider theme={currentTheme}>{children}</ThemeProvider>
       </DocsContainer>
       <ThemeProvider theme={currentTheme}>
-        <TableOfContents />
+        <TableOfContents hidden={!!hideToc} />
```

Comment:
Passing `hideToc` as a prop because we want `TableOfContents` to conditionally hide its contents - but not its container. We still want an empty container because the container affects the page layout (pushing the main content to the left side – see screenshot above), and we want the layout to be consistent across pages.